packages/page-claims/src/statements/polkadot/regular.tsx

```diff
@@ -0,0 +1,144 @@
+// Copyright 2017-2020 @polkadot/app-claims authors & contributors
```

Comment:
Remove, rather than duplicate - point to ipfs (we have the hash, default) and website the website. (Better than duplicating the html and even md for it)

Reply:
I think we legally have the obligation to show the statement on the page where the attestation is made. Can you confirm @lsaether ?

Reply:
Yes the statements need to appear on the page directly.

Reply:
We had a little discussion with Amaury to day we implemented that. We chose to duplicate it rather than fetching it from the website, because:
- it won't change
- it removes a dependency to this website

But it's ultimately up to you @jacogr if you prefer the content of the statement to be fetched.

Reply:
I indicated before that this whole React component for it is just horrible. We are duplicating and adding bloat for no reason, at the very least it is not a rework of the text like we have. 

Since we sign the hash, the preferred approach is always going to be o "point to the hash". If there is a legal reason not to, well, the hash of what we sign and what we display is not the same since we reworked. They need to match.

Reply:
@jacogr there's no silver bullet. Please tell us which one you want between 1, 2, 3 or 4:
1. Keep as-is. As you said, it's not matching.
2. Use md. Cleaner, but it's also not matching. Plus we need to do html->md conversion.
3. Use `fetch()`. We will get back a HTML file, there will be some ugly code to get the innerHTML of the `body`, and probably some `dangerouslySetInnerHTML`. Plus latency for user.
4. Use iframe. We lose control over styling.

We're happy with any solution.

Reply:
Well, we are out of time :) I would prefer 4. It is the original, no tweaks.

Reply:
iframe done.

> Well, we are out of time

How would you do it if we had time?

Reply:
Simple answer - without too many interruptions from me. :)

1. Skip local non-matching
2. Allow opening in with target="_blank"` ipfs (with matching hash) and polkadot (cannot check hash 100%)

Not going to fight this, since it needs to go.